v2/jdbc-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/JdbcToSpannerOptions.java

```diff
@@ -0,0 +1,187 @@
+      example = "gs://your-bucket/driver_jar1.jar,gs://your-bucket/driver_jar2.jar")
+  String getDriverJars();
+
+  void setDriverJars(String driverJar);
+
+  @TemplateParameter.Text(
+      order = 2,
+      optional = false,
+      regexes = {"^.+$"},
+      groupName = "Source",
+      description = "JDBC driver class name",
+      helpText = "The JDBC driver class name.",
+      example = "com.mysql.jdbc.Driver")
+  String getDriverClassName();
+
+  void setDriverClassName(String driverClassName);
+
+  @TemplateParameter.Text(
+      order = 3,
+      optional = false,
+      regexes = {
+        "(^jdbc:[a-zA-Z0-9/:@.?_+!*=&-;]+$)|(^([A-Za-z0-9+/]{4}){1,}([A-Za-z0-9+/]{0,3})={0,3})"
+      },
+      groupName = "Source",
+      description = "JDBC connection URL string.",
+      helpText =
+          "The JDBC connection URL string. For example, `jdbc:mysql://some-host:3306/sampledb`. Can"
+              + " be passed in as a string that's Base64-encoded and then encrypted with a Cloud"
+              + " KMS key. Note the difference between an Oracle non-RAC database connection string"
+              + " (`jdbc:oracle:thin:@some-host:<port>:<sid>`) and an Oracle RAC database"
+              + " connection string (`jdbc:oracle:thin:@//some-host[:<port>]/<service_name>`).",
+      example = "jdbc:mysql://some-host:3306/sampledb")
+  String getConnectionURL();
+
+  void setConnectionURL(String connectionURL);
+
+  @TemplateParameter.Text(
+      order = 4,
+      optional = true,
+      regexes = {"^[a-zA-Z0-9_;!*&=@#-:\\/]+$"},
+      groupName = "Source",
+      description = "JDBC connection property string.",
+      helpText =
+          "Properties string to use for the JDBC connection. Format of the string must be"
+              + " [propertyName=property;]*.",
+      example = "unicode=true;characterEncoding=UTF-8")
+  String getConnectionProperties();
+
+  void setConnectionProperties(String connectionProperties);
+
+  @TemplateParameter.Text(
+      order = 5,
+      optional = true,
+      regexes = {"^.+$"},
+      groupName = "Source",
+      description = "JDBC connection username.",
+      helpText =
+          "The username to be used for the JDBC connection. Can be passed in as a Base64-encoded"
+              + " string encrypted with a Cloud KMS key.")
+  String getUsername();
+
+  void setUsername(String username);
+
+  @TemplateParameter.Password(
+      order = 6,
+      optional = true,
+      groupName = "Source",
+      description = "JDBC connection password.",
+      helpText =
+          "The password to be used for the JDBC connection. Can be passed in as a Base64-encoded"
+              + " string encrypted with a Cloud KMS key.")
+  String getPassword();
+
+  void setPassword(String password);
+
+  @TemplateParameter.Text(
+      order = 7,
+      optional = true,
+      groupName = "Source Parameters",
+      description = "The name of a column of numeric type that will be used for partitioning.",
```

Comment:
Is this just a single column name or tableName:columnName combination? Will help to give example. If it is just single column name - then how is mapping done to table, since multiple tables can have different partitioning column.

Reply:
In the first version, this will work only for a single table. In subsequent versions, support for running this for multiple tables using a schema mapping file will be added and these options will be deprecated.